### DISCUSSION TITLE
CvInvoke.FindChessboardCorners() throws Unable to find an entry point named 'cveOutputArrayFromVectorOfPoint' in DLL 'cvextern'.

### DISCUSSION THREAD

**Discussion by argoallik** [emgucv/emgucv#554] [Q&A]
`var image = new Image<Gray, byte>(imageFile);`
`var cornerPoints = new VectorOfPoint();`
`Size patternSize = new Size(9, 7);`
`bool result = CvInvoke.FindChessboardCorners(image, patternSize, cornerPoints);`

Thows exception: 
'System.EntryPointNotFoundException' in Emgu.CV.Platform.NetStandard.dll
An unhandled exception of type 'System.EntryPointNotFoundException' occurred in Emgu.CV.Platform.NetStandard.dll
Unable to find an entry point named 'cveOutputArrayFromVectorOfPoint' in DLL 'cvextern'.

Stack trace:
at Emgu.CV.Util.VectorOfPoint.cveOutputArrayFromVectorOfPoint(IntPtr vec)
at Emgu.CV.Util.VectorOfPoint.GetOutputArray()
at Emgu.CV.CvInvoke.FindChessboardCorners(IInputArray image, Size patternSize, IOutputArray corners, CalibCbType flags)
at OpenCVCalibration.Program.Main(String[] args) in C:...\OpenCVCalibration\Program.cs:line 42

Emgu.CV and Emgu.CV.runtime.windows packages are both version 4.5.3.4721
OS is Windows 10 Pro 10.0.19042 build 19042

**Comment by emgucv** [comment]
Test with release package "libemgucv-windesktop-4.5.3.4721.zip". 

Replaced Emgu.CV.Example\HelloWorld\Program.cs with the following code:

```
using System;
using System.Drawing;
using Emgu.CV;
using Emgu.CV.CvEnum;
using Emgu.CV.Structure;
using Emgu.CV.Util;

namespace HelloWorld
{
    class Program
    {
        static void Main(string[] args)
        {
            CvInvoke.Init();
            var image = new Image<Gray, byte>(640, 480);
            var cornerPoints = new VectorOfPoint();
            Size patternSize = new Size(9, 7);
            bool result = CvInvoke.FindChessboardCorners(image, patternSize, cornerPoints);
        }
    }
}

```
The test program run through without any issues on Windows 10 Pro.

**Reply by argoallik** [reply]
Thank you. I found the problem. In my case it was that in my project I was targeting .net framework and I had ANY CPU as build platform. Changing it to x64 solved this problem. It is also mentioned in documentation.
